build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:3.6.0'
```

Comment:
I confirmed my android studio was able to build the project with this updated version